### DISCUSSION TITLE
Override Radio.Group in custom theme

### DISCUSSION THREAD

**Discussion by telmen** [mantinedev/mantine#2558] [Q&A]
Hello,

Is it possible to override `Radio.Group` in the `components` field in theme customisation? Tried `'Radio.Group'` already. Thanks!

**Comment by rtivital** [comment]
You can add default props to it with `RadioGroup`, if you want to change styles apply them to InputWrapper

**Reply by robinclaes** [reply]
Hi @rtivital , this is not working for me. Can you indicate what I am doing wrong? 
It is an entry in the `components` property of the theme override.

```
RadioGroup: {
    defaultProps: {
      labelProps: {
        color: 'pink',
        fontWeight: 'bold',
      },
      label: 'This is not working.',
    },
}
```

**Reply by rtivital** [reply]
https://codesandbox.io/s/withered-violet-9m4sw9?file=/src/App.tsx
